packages/loctool/test/Project.test.js

```diff
@@ -19,31 +19,52 @@
 
 var fs = require('fs');
 
-if (!Project) {
-    var ProjectFactory = require("../lib/ProjectFactory.js");
-    var Project = require("../lib/Project.js");
-}
+var ProjectFactory = require("../lib/ProjectFactory.js");
+var Project = require("../lib/Project.js");
 
 function rmrf(path) {
     if (fs.existsSync(path)) {
         fs.unlinkSync(path);
     }
 }
 
-function diff(a, b) {
-    var min = Math.min(a.length, b.length);
-
-    for (var i = 0; i < min; i++) {
-        if (a[i] !== b[i]) {
-            console.log("Found difference at character " + i);
-            console.log("a: " + a.substring(i));
-            console.log("b: " + b.substring(i));
-            break;
-        }
-    }
-}
-
 describe("project", function() {
+    afterEach(function() {
+        [
+            "./test/testfiles/loctest-extracted.xliff",
+            "./test/testfiles/loctest-new-es-US.xliff",
+            "./test/testfiles/loctest-new-ja-JP.xliff",
+            "./test/testfiles/loctest-new-zh-Hans-CN.xliff",
+            "./test/testfiles/es-US/md/test1.md",
+            "./test/testfiles/ja-JP/md/test1.md",
+            "./test/testfiles/zh-Hans-CN/md/test1.md",
+            "./test/testfiles/loctest-new-en-GB.po",
+            "./test/testfiles/loctest-new-en-GB.xliff",
+            "./test/testfiles/project2/loctest-extracted.xliff",
+            "./test/testfiles/project2/loctest-new-es-US.xliff",
+            "./test/testfiles/project2/loctest-new-ja-JP.xliff",
+            "./test/testfiles/project2/loctest-new-ru-RU.xliff",
+            "./test/testfiles/project2/loctest-new-en-GB.xliff",
+            "./test/testfiles/project2/loctest.xliff",
+            "./test/testfiles/translations/loctest.xliff",
+            "./test/testfiles/project3/loctest-extracted.xliff",
+            "./test/testfiles/project3/loctest-new-es-US.xliff",
+            "./test/testfiles/project3/loctest-new-ja-JP.xliff",
+            "./test/testfiles/project3/loctest-new-zh-Hans-CN.xliff",
+            "./test/testfiles/project3/es-US.mock",
+            "./test/testfiles/project3/ja-JP.mock",
+            "./test/testfiles/project3/zh-Hans-CN.mock",
+            "./test/testfiles/loctest-extracted.po",
+            "./test/testfiles/loctest-new-es-US.po",
+            "./test/testfiles/loctest-new-ja-JP.po",
+            "./test/testfiles/loctest-new-zh-Hans-CN.po",
+            "./test/testfiles/project3/loctest-extracted.po",
+            "./test/testfiles/project3/loctest-new-es-US.po",
+            "./test/testfiles/project3/loctest-new-ja-JP.po",
+            "./test/testfiles/project3/loctest-new-zh-Hans-CN.po"
+        ].forEach(rmrf);
```

Comment:
nit: Personally, I feel like it was better before when each test took care of its own files.

Reply:
Hah, those ARE the files that that particular file full of tests produces! It was a prolific polluter. Also, after I rebase, it will get the results of the test file clean-up PR I did last week...

Reply:
Okay, after the rebase, things are straightened out here.